ember-modifier/src/-private/function-based/modifier.ts

```diff
@@ -121,7 +121,8 @@ export default function modifier(
     element: Element,
     positional: unknown[],
     named: object
-  ) => void | Teardown
+  ) => void | Teardown,
+  name?: string
```

Comment:
@patricklx as this is a new argument, I guess we should document it in readme to ensure ppl know it exists, how to use and what is the purpose.

@patricklx additionally, v3.2.0+ of this package accepted second argument as an object to aid in v3 -> v4 migrations as described in https://github.com/ember-modifier/ember-modifier/blob/main/MIGRATIONS.md#function-based-modifiers

I'm thinking if we should make function signature like

```
function modifier(
  fn: (
    element: Element,
    positional: unknown[],
    named: object
  ) => void | Teardown,
  options?: { name: string }
```

so that second argument may be expanded again in future if more options can/need to be passed in

Reply:
done